src/CoreBundle/private/library/functions/callback_functions/gcf_CMSUserWithImage.fun.php

```diff
@@ -9,9 +9,12 @@
  * file that was distributed with this source code.
  */
 
+/**
+ * @deprecated since 6.3.0
```

Comment:
Could you add a short note on what to do instead (if there's a replacement) or a "no longer used" text?

The function is still used in `TAdbCmsUser::GetDisplayValue`. If we mark something as deprecated, we should try to remove it from Chameleon code.

Reply:
Oh, i did not check database generated code here. So, to allow a deprecation i moved the code where it belongs to the TCMSListManagerCMSUser class and added deprecation todos.